hotham-simulator/src/state.rs

```diff
@@ -196,25 +202,38 @@ impl State {
         let mut x_rot = 0.0;
         let mut y_rot = 0.0;
 
-        match self.event_rx.as_ref()?.try_recv().ok()? {
-            HothamInputEvent::KeyboardInput { key } => match key? {
+        let now = Instant::now();
```

Comment:
nit: a couple of comments explaining the logic could be helpful here

Reply:
added a couple of small comments but can always add more if you'd like

Reply:
This is good enough for now! In general for these sorts of things a docstring above the function is nice but it's not critical here